docs/cristin-proxy-swagger.yaml

```diff
@@ -3090,6 +3090,40 @@ components:
                 description: Two letter country code (ISO 3166-1 alpha-2)
               label:
                 $ref: '#/components/schemas/Languages'
+        awards:
+          type: array
+          items:
+            type: object
+            properties:
+              title:
+                type: string
+                description: Title of the award
+              year:
+                type: integer
+                description: Year of the award
+              type:
+                type: object
+                properties:
+                  type:
+                    enum:
+                      - FORSKFORMIDL
+                      - FORSKARBEID
+                    description: The code for this type of activity. Can be FORSKFORMIDL (Dissemination of research results) or FORSKARBEID (Research activity)
```

Comment:
What do you think @brinxmat ? Keep the enums as they are in Cristin or call them something else here in proxy?

Reply:
ResearchDissemination
Research

Reply:
Done